src/Symfony/Component/Validator/Constraints/Url.php

```diff
@@ -48,13 +53,15 @@ public function __construct(
         ?callable $normalizer = null,
         ?array $groups = null,
         mixed $payload = null,
+        ?bool $requireTld = null,
     ) {
         parent::__construct($options, $groups, $payload);
 
         $this->message = $message ?? $this->message;
         $this->protocols = $protocols ?? $this->protocols;
         $this->relativeProtocol = $relativeProtocol ?? $this->relativeProtocol;
         $this->normalizer = $normalizer ?? $this->normalizer;
+        $this->requireTld = $requireTld ?? $this->requireTld;
```

Comment:
In the issue, we discussed deprecating not setting this, so we can change the default to `true` (which is in sync with the `Hostname` constraint's `requireTld` option).

Do we still want to do this?

If we don't, I would say we change the default of the parameter to `false` instead (I see no point making it nullable?)

Reply:
see #54588